sdk/include/opentelemetry/sdk/resource/semantic_conventions.h

```diff
@@ -13,6 +13,19 @@
 #include "opentelemetry/common/macros.h"
 #include "opentelemetry/version.h"
 
+/*
+ * Special considerations for HostArchValues
+ *
+ * The sys/systemcfg.h header on AIX is known to define an IA64 macro,
+ * which collides with HostArchValues::IA64.
```

Comment:
This work around is no longer necessary, after:

* #1761

>  Change naming to use camel case, to avoid collisions with macros.

Reply:
I will change the PR to withdraw this workaround